src/resources/uart_esp32.cc

```diff
@@ -636,8 +650,13 @@ uint32 UartResourceGroup::on_event(Resource* r, word data, uint32 state) {
       reinterpret_cast<UartResource*>(r)->clear_tx_event_in_queue();
       break;
 
+    case UART_FIFO_OVF:
+    case UART_BUFFER_FULL:
+      reinterpret_cast<UartResource*>(r)->rx_buffer()->signal_dropped_data();
+      [[fallthrough]];
     default:
       state |= kErrorState;
+      reinterpret_cast<UartResource*>(r)->increment_errors();
```

Comment:
Is wrapping around to zero useful? Maybe we should just avoid incrementing it after a certain point?

Reply:
I switched to a int64.
No reason to worry about corner cases.
done.